fhir-persistence-blob/src/test/java/com/ibm/fhir/persistence/blob/async/ReactorTests.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * (C) Copyright IBM Corp. 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ 
+package com.ibm.fhir.persistence.blob.async;
+
+import static org.testng.Assert.fail;
+
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.reactivestreams.Subscriber;
+import org.reactivestreams.Subscription;
+import org.testng.annotations.Test;
+
+import reactor.core.publisher.BaseSubscriber;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+/**
+ * Tests using Flux and Mono which are used in the Azure SDK for async
+ * handling.
+ */
+public class ReactorTests {
```

Comment:
just a playground for this stuff or tests we actually want?

Reply:
Playground...but I want to keep this because it's useful from an educational perspective as well as making sure if you change anything, you can check the fundamentals work as you expect.

Reply:
works for me